packages/nextjs/scaffold.config.ts

```diff
@@ -26,6 +27,15 @@ const scaffoldConfig = {
   // It's recommended to store it in an env variable:
   // .env.local for local testing, and in the Vercel/system env config for live apps.
   walletConnectProjectId: process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID || "3a8170812b534d0ff9d794f19a901d64",
+
+  contracts: {
+    _SandGardenStreamsOld: {
+      fromBlock: 101114639,
```

Comment:
Not sure what's the place to add this 😅 just wanted this value to be shared with others cloning the repo

Reply:
This is great! I was using env vars, so you can have different values on different environments/chains... but I think this is more straight-forward.

Made a few tweaks here b43a70c4f7b1f3be61e03a2ff271fa06edd0096b